packages/shared/src/components/search/SearchPanel/common.tsx

```diff
@@ -8,3 +13,20 @@ export const providerToLabelTextMap: Record<SearchProviderEnum, string> = {
   [SearchProviderEnum.Posts]: 'Search posts',
   [SearchProviderEnum.Chat]: 'Ask daily.dev AI',
 };
+
+export const providerToIconMap: Record<
+  SearchProviderEnum,
+  FunctionComponent<IconProps>
+> = {
+  [SearchProviderEnum.Posts]: SearchIcon,
+  [SearchProviderEnum.Chat]: ({ className, ...rest }: IconProps) => (
+    <MagicIcon
+      className={classNames(
+        className,
+        'rounded-6 bg-gradient-to-t from-onion-40 to-cabbage-40 p-0.5 text-white',
```

Comment:
The color here should be named variants. (not sure if we have these exact already, but would think so? (generally it's always "wrong" to use the .50 directly basically.

Sub note: the p-0.5 seems tiny?

Reply:
Sorted colors on epic already. As for 0.5, it is like that in design, guess because icons are small:
<img width="252" alt="image" src="https://github.com/dailydotdev/apps/assets/9803078/20817e49-a87c-475b-91a0-72f3ecd23881">